pkg/virt-operator/install-strategy/create_test.go

```diff
@@ -829,46 +782,31 @@ var _ = Describe("Create", func() {
 			virtClient.EXPECT().SecClient().Return(secClient).AnyTimes()
 		})
 
+		executeTest := func(scc *secv1.SecurityContextConstraints, expectedPatch string) {
```

Comment:
Here you could probably use a `DescribeTable` too...

Reply:
@danielBelenky Thank you. Done.